spec/truefi2/BorrowingMutex.spec

```diff
@@ -0,0 +1,36 @@
+methods {
+    isUnlocked(address) returns bool envfree
+    isBanned(address) returns bool envfree
+}
+
+definition isReasonableAddress(address alice) returns bool = alice != 0 && alice != 1;
+definition isReasonableEnv(env e) returns bool = isReasonableAddress(e.msg.sender);
+
+rule onePlusTwoEqualsThree(uint one, uint two) {
```

Comment:
This is left from previous commit for sake of an example, will be removed later.